src/index.html

```diff
@@ -3,12 +3,56 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="header-logo">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav>
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a href="#" class="nav-link is-active">Apple</a>
+          </li>
+
+          <li class="nav-item">
+            <a href="#" class="nav-link">Samsung</a>
+          </li>
+
+          <li class="nav-item">
+            <a href="#" class="nav-link">Smartphones</a>
+          </li>
+
+          <li class="nav-item">
+            <a href="#" class="nav-link"
+            data-qa="hover">Laptops & Computers
+            </a>
```

Comment:
```suggestion
            <a
              href="#"
              class="nav-link"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```
correct formatting - FYI (take it into account in the next tasks)